src/main/java/duke/main/Duke.java

```diff
@@ -0,0 +1,179 @@
+package duke.main;
+
+import duke.util.Parser;
```

Comment:
try to import in lexicographically order so like import duke.exception before duke.util